### ISSUE TITLE
A11y: skipnav absent on some example site pages

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#727]
An a11y best practice is to have a skip nav to jump keyboard users to main content.  Some pages on the example site are missing skipnav links.

> **ESTIMATE** 1
